appbuilder/services/npm-service.ts

```diff
@@ -93,6 +145,21 @@ export class NpmService implements INpmService {
 		}).future<any>()();
 	}
 
+	public isScopedDependency(dependency: string): boolean {
+		let matches = dependency.match(this.scopedDependencyRegExp);
+
+		return !!(matches && matches[0]);
+	}
+
+	public getScopedDependencyInformation(dependency: string): IScopedDependencyInformation {
+		let matches = dependency.match(this.scopedDependencyRegExp);
+
+		return {
+			name: matches[1],
```

Comment:
You should probably check if this is in fact a scoped dependency by calling `this.isScopedDependency` beforehand. Otherwise you might end up with `matches` being undefined and a null reference here